Dockerfile

```diff
@@ -1,8 +1,6 @@
 # Build Postgres
-FROM neondatabase/rust:1.58 AS pg-build
-WORKDIR /pg
-
-USER root
+FROM 369495373322.dkr.ecr.eu-central-1.amazonaws.com/rust:pinned AS pg-build
```

Comment:
How is that connected to our old `neondatabase/rust:1.58` images?
Would I still be able to push changes to https://github.com/neondatabase/docker-images and get the updated image?

Reply:
They are closely related to the new images of rust & rustlegacy https://github.com/neondatabase/build/blob/main/rust/Dockerfile & https://github.com/neondatabase/build/blob/main/rustlegacy/Dockerfile

Reply:
Looks like the repo I've linked is obsolete now, if the new ones are used.

Would be nice to hear later a digest on our recent CI and infra changes, that seems to be big at least.